ruby-on-rails/ruby-on-rails-quiz.md

```diff
@@ -685,9 +686,9 @@ category.name = 'News'
 saved_name = _____
 ```
 
-- [ ] category.name_was
+- [x] category.name_was
 - [ ] category.saved(:name)
-- [x] category.changes[:name]
+- [ ] category.changes[:name]
```

Comment:
I went as back as Rails 3.0.9 documentation and could not find a place where `model_instance.changes[attribute.to_sym]` would ever return a String instead of a hash containing attributes as keys and pairs as [old_value, new_value].

```ruby
person.changes # => {}
person.name = 'bob'
person.changes # => { 'name' => ['bill', 'bob'] }
```